build/build-gh-pages.js

```diff
@@ -1,5 +1,30 @@
 const fs = require("fs").promises;
 
+
+// This object exist only to format the HTML as designed
+// type: warning-type as stored as filename in design/svg-folder
+// title: title that will be desplayed in html
+// color: wich colored svg to look for
+// Order is based on design i figma
+const warningTypes = [
+  {type: 'avalanches', title: 'Avalanches', colors: ['yellow', 'orange', 'red']},
+  {type: 'landslide', title: 'Landslide', colors: ['yellow', 'orange', 'red']},
+  {type: 'flood', title: 'Flood', colors: ['yellow', 'orange', 'red']},
+  {type: 'wind', title: 'Gust/Wind', colors: ['yellow', 'orange', 'red']},
+  {type: 'wind', title: 'Gale', colors: ['yellow']},
+  {type: 'rain', title: 'Rain', colors: ['yellow', 'orange', 'red']},
+  {type: 'rainflood', title: 'Rain Flood', colors: ['yellow', 'orange']},
+  {type: 'lightning', title: 'Lightning', colors: ['yellow']},
+  {type: 'snow', title: 'Snow', colors: ['yellow', 'orange', 'red']},
+  {type: 'snowstorm', title: 'Snow Storm', colors: ['yellow']},
+  {type: 'icing', title: 'Ice', colors: ['yellow', 'orange', 'red']},
+  {type: 'stormsurge', title: 'Stormsurge', colors: ['yellow', 'orange', 'red']},
+  {type: 'polarlow', title: 'Polarlow', colors: ['yellow', 'orange']},
+  {type: 'forestfire', title: 'Forestfire', colors: ['yellow', 'orange']},
+  {type: 'generic', title: 'Ising', colors: ['yellow', 'orange']}
```

Comment:
`icon-warning-drivingconditions` and `icon-warning-ice` are no longer included in the documentation. Is that intentional?

Reply:
The selection of icons is based on these three:

https://www.figma.com/file/kSKhVEehmQx4YgEQskZmQZ/Icons?node-id=158%3A124
https://hjelp.yr.no/hc/no/articles/360014052634
https://docs.google.com/spreadsheets/d/1EvgHGeNnyx054IobPSr8fHG2zwJzzRUs6XD3moSp1kU/edit#gid=1634028568

Reply:
I asked for more information in https://nrknyemedier.atlassian.net/browse/YR-5365 now.